index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <div class="container">
+  <h1 id = "title">freeCodeCamp Survey Form</h1>
+<p id = "description">Thank you for taking the time to help us improve the platform</p>
+<form id="survey-form">
+  <label for="name" id = "name-label">
+    Name
+    <input type="text" id="name" required placeholder="Enter your name">
+  </label>
+  <br>
+  <label for="name" id = "email-label" >
+    Email
+    <input id = "email" type = "email"required placeholder="Enter your email">
+  </label>
+  <br>
+  <label for="number" id = "number-label">
+    Age (optional)
+    <input id="number" type="number" min ="18" max = "90" placeholder="Age">
+  </label>
+  <br>
+  <label for="dropdown">Which option best describes your current role?</label>
+
+<select name="dropdown" id="dropdown">
+  <option value="student">Student</option>
+  <option value="full time job">Full time job</option>
+</select>
+<br>
+  <label>
+    <p>Would you recommend freeCodeCamp to a friend?</p>
+    <input
+    name="user-recommend"
+    value="definitely"
+    type="radio"
+    class="input-radio"
+    checked
+    />Definitely</label>
+    <label>
+    <input
+    name="user-recommend"
+    value="Maybe"
+    type="radio"
+    class="input-radio"
+    checked
+    />Maybe</label>
+    <label>
+    <input
+    name="user-recommend"
+    value="Not sure"
+    type="radio"
+    class="input-radio"
+    checked
+    />Not sure</label>
+    <label>
+    <br>
+<div class="form-group">
```

Comment:
- [ ]  This **div** also can be another semantic tag like **section**

Reply:
As I mentioned above, Kindly, I suggest you use polite and clear comments, like:

```
This is a polite suggestion: "I think it would be better to use semantic tags"

This reads as an order: "Use semantic tags"
```

Reply:
Please, make sure your comment is clear and explicit.
Kindly, I suggest you update this comment. 😄

Reply:
Good job pointing out a good reason to improve the code. 👍🏿 
- [ ] Kindly give reasons why you are making your suggestions as it will go a long way to make your suggestions more meaningful. 🤝 

In this situation, you could explain why it is important to use semantic tags rather than just using a `<div>`.
Example: `It is good practice to use semantic tags because it ensures standardization and  consistency in your codebase` 🎯